packages/loader/container-utils/src/error.ts

```diff
@@ -77,32 +79,43 @@ export class DataCorruptionError extends LoggingError implements IErrorBase {
     }
 }
 
-export class DataProcessingError extends LoggingError implements IErrorBase {
+export class DataProcessingError extends LoggingError implements IErrorBase, IFluidErrorBase {
     readonly errorType = ContainerErrorType.dataProcessingError;
     readonly canRetry = false;
 
-    constructor(errorMessage: string, props?: ITelemetryProperties) {
+    constructor(
+        errorMessage: string,
```

Comment:
it seems easy to confuse errorCode and message here, as they are both strings. i wonder if we want to take an object which is more explicit. not need to change. just a thought

Reply:
You mean for constructor args?  Good comment.  WF for now.